src/usr/local/www/diag_packet_capture.php

```diff
@@ -512,7 +518,15 @@ function hostport_array_fixer($items,$type) {
 	}
 
 	if (in_array($proto, $protos)) {
-		$matches[] = fixup_not(str_replace('carp', 'proto 112', $proto));
+		switch ($proto) {
```

Comment:
I think I'd rather see this have a `break` in each case and then add to `$matches` outside the `switch`. This way works, but is a bit awkward to follow and unnecessarily runs the extra lower steps. Even if they are not matching, it's still doing more than it needs to.

Reply:
done